app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,13 @@
+<% provide(:title, "About") %>
+<h1>About</h1>
+<p>
+  The <a href="https://www.railstutorial.org/"><em>Ruby on Rails
+  Tutorial</em></a>, part of the
```

Comment:
using link_to instead of a tag
with link address, you should research gem config to manage them 
https://github.com/rubyconfig/config